hashin.py

```diff
@@ -284,41 +284,45 @@ def main():
     import argparse
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        'package',
-        help="package (e.g. some-package==1.2.3 or just some-package)"
+        'packages',
+        help="One or more package specifiers (e.g. some-package or some-package==1.2.3)",
+        nargs='+'
     )
     parser.add_argument(
-        'requirements_file',
+        '-r', '--requirements-file',
         help="requirements file to write to (default requirements.txt)",
-        default='requirements.txt', nargs='?'
+        default='requirements.txt'
     )
     parser.add_argument(
-        'algorithm',
+        '-a', '--algorithm',
         help="The hash algorithm to use: one of sha256, sha384, sha512",
-        default='sha256', nargs='?'
+        default='sha256'
     )
     parser.add_argument(
-        "-v, --verbose",
+        '-v', '--verbose',
         help="Verbose output",
         action="store_true",
-        dest='verbose',
     )
     parser.add_argument(
-        '-p, --python-version',
+        '-p', '--python-version',
         help='Python version to add wheels for. May be used multiple times.',
         action='append',
         default=[],
-        dest='python_version',
     )
 
     args = parser.parse_args()
-    return run(
-        args.package,
-        args.requirements_file,
-        args.algorithm,
-        args.python_version,
-        verbose=args.verbose,
-    )
+
+    ret = 0
+    for package in args.packages:
+        ret = ret | run(
```

Comment:
Too much logic here in the `if __name__==__main__` block thing. Can you change `run()` to take a list instead. 
And it, should return 0 if ALL worked. But if there's any exceptions (e.g. HTTP 404 on an unknown package the exception itself will take care of making sure it doesn't exit with a 0.